_progress-bar.scss

```diff
@@ -0,0 +1,279 @@
+// progress-bar
+// ===
+
+// Configurable Variables
+// ---
+
+$progress-bar__progress-color: black !default;
+$progress-bar__success-color: $success-color !default;
+$progress-bar__error-color: $error-color !default;
+$progress-bar__background-color: $neutral-50 !default;
+
+
+// Dependencies
+// ---
+
+// Override Font Awesome's default path
+$fa-font-path: '/bower_components/font-awesome/fonts';
+
+@import 'stencil-variables/variables';
+@import 'level/level';
+@import 'font-awesome/scss/font-awesome';
+@import 'stencil-utils/dist/visibility/visibility';
+
+
+// Progress Bar root
+// ---
+
+.c-progress-bar {
+    display: flex;
+    align-items: center;
+    height: $tap-size;
+    margin: 0 15px;
+}
+
+
+
+// Radial Progress Bar
+// ---
+
+.c-progress-bar__radial {
+    &:not(:empty) {
+        position: relative;
+
+        width: 45px;
+        height: 45px;
+        margin-right: 10px;
+    }
+}
+
+// Radial Progress Bar Background
+// ---
+
+.c-progress-bar__radial-background {
+    stroke: $progress-bar__background-color;
+}
+
+// Radial Progress Bar Progress
+// ---
+//
+// Used to show the current level of progress
+
+.c-progress-bar__radial-progress {
+    .c-progress-bar.c--progress & {
+        stroke: $progress-bar__progress-color;
+    }
+
+    .c-progress-bar.c--success & {
+        stroke: $progress-bar__success-color;
+    }
+
+    .c-progress-bar.c--error & {
+        stroke: $progress-bar__error-color;
+    }
+}
+
+// Radial Progress Bar SVG
+// ---
+//
+// Where the magic happens
+
+.c-progress-bar__svg {
+    .c-progress-bar.c--direction-rtl & {
+        transform: scaleX(-1);
+    }
+}
+
+
+
+// Progress Bar Middle
+// ---
+//
+// Holds the linear progress bar and the label
+// Needed as a separate element so we can flexbox everything nicely
+
+.c-progress-bar__middle {
+    flex-grow: 1;
+    margin-right: 5px;
+}
+
+
+
+// Linear Progress Bar Background
+// ---
+
+.c-progress-bar__linear-background {
+    position: relative;
```

Comment:
I generally recommend that a comment be added to explain the intended purpose for `position` properties.

It's not always obvious why these are here, or which elements/classes they are meant to interact with. Something like... `Position relative, so the child .c-progress-bar__linear-progress element can be absolutely positioned` and or any other reasons.